content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md

```diff
@@ -59,11 +59,22 @@ Additionally, kubeadm informs the user if the certificate is externally managed;
 `kubelet.conf` is not included in the list above because kubeadm configures kubelet for automatic certificate renewal.
 {{< /note >}}
 
+{{< warning >}}
+On nodes created with `kubeadm init`, prior to kubeadm version 1.17, there is a
+[bug](https://github.com/kubernetes/kubeadm/issues/1753) where you manually have to modify the contents of `kubelet.conf`. After `kubeadm init` finishes, you should update `kubelet.conf` to point to the
```

Comment:
I think we should specify that this problem happens only on the node where kubeadm init is executed

Reply:
@fabriziopandini 
i think the warning already clarifies that:
> On nodes created with `kubeadm init`
?